src/common/sentry/sentry.filter.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  Catch,
+  ArgumentsHost,
+  HttpException,
+  HttpStatus,
+} from '@nestjs/common';
+import { BaseExceptionFilter } from '@nestjs/core';
+import * as Sentry from '@sentry/nestjs';
+
+@Catch()
+export class SentryFilter extends BaseExceptionFilter {
+  /**
+   * Exception handling and Sentry integration
+   */
+  catch(exception: unknown, host: ArgumentsHost) {
+    try {
+      const ctx = host.switchToHttp();
+      const request = ctx.getRequest();
+
```

Comment:
@rabiajaved31 Please check the latest dev in motac-mobile-api repo and pull this again, I have added a method here to ignore 404 api call errors.